xcodeproj/internal/bazel_integration_files/bazel_build.sh

```diff
@@ -202,3 +209,32 @@ for output_group in "${output_groups[@]}"; do
     exit 1
   fi
 done
+
+# Async actions
+#
+# For these commands to run in the background, both stdout and stderr need to be
+# redirected, otherwise Xcode will block the run script.
+
+log_dir="$OBJROOT/rules_xcodeproj_logs"
+mkdir -p "$log_dir"
+
+# Report errors from previous async actions
+shopt -s nullglob
+for log in "$log_dir"/*.async.log; do
+  if [[ -s "$log" ]]; then
+    command=$(basename "${log%.async.log}")
+    echo "warning: Previous run of \"$command\" had output:" >&2
+    sed "s|^|warning: |" "$log" >&2
+    echo "warning: If you believe this is a bug, please file a report here:" \
+"https://github.com/buildbuddy-io/rules_xcodeproj/issues/new?template=bug.md." \
```

Comment:
```suggestion
"https://github.com/buildbuddy-io/rules_xcodeproj/issues/new?template=bug.md" \
```

The trailing period broke the template

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/pull/1041